tutorials/power_distribution/api.h

```diff
@@ -0,0 +1,14 @@
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+gpio_t DASHBOARD_POWER = PB0;
+gpio_t SERVICE_SECTION_POWER = PB1;
+gpio_t THROTTLE_POWER = PB2;
+
+void power_device(gpio_t pin);
+
+void power_all();
```

Comment:
If there are no arguments, you should specify `void` as the argument:

```c
void power_all(void)
```

Same below and in `power_distribution.c`.